### PR TITLE
added pretraining loss

### PR DESCRIPTION
**Pull request by ryanwongsa**

### CODE REVIEW

**Comment by ryanwongsa** [comment]
Training Loss:
example_training_loss
Validation F1 Score:
validation_f1score
